code/Language/Drasil/Document.hs

```diff
@@ -115,7 +119,24 @@ instance HasShortName  Contents where
     "Bibliography list of references cannot be referenced. " ++
     "You must reference the Section or an individual citation."
 
-
+---------------------------------------------------------------------------
+-- smart constructors needed for LabelledContent
+-- nothing has a shortname right now
+mkTableLC :: String -> String -> String -> Contents -> LabelledContent
+mkTableLC uidLC labelUID refAdd tbl = llcc uidLC 
+  (mkLabelRA labelUID refAdd {- table ^. refAdd? -}) tbl Nothing
```

Comment:
1. Since `Table` already has a `RefAdd`, perhaps we can pull out the reference address to pass into `mkLabelRA`?

2. Another way this can be approached is by removing `RefAdd` from line 74 of this file, and making that another argument that's passed into `mkTableLC`.

3. Also, is the current implementation good, or was the following more what was in mind for these constructors: 
```
mkTableLC uidLC labelUID refAdd header data title show = llcc uidLC  (mkLabelRA labelUID refAdd) (Table header data title show) Nothing
```
Asking because the examples currently build tables like https://github.com/JacquesCarette/Drasil/blob/8885c9c960514948f0becdccf26a83144c21a03b/code/Example/Drasil/GamePhysics/Body.hs#L695-L700, which I'm unsure of whether is desirable or not...

Reply:
1. We should have a single `RefAdd` for tables. But see below.
2. I think that is a good idea
3. and your alternate implementation seems best.